README.md

```diff
@@ -0,0 +1,72 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Portfolio of Microverse projects
+
+![screenshot](images/portfolio.png)
+
+This project is a portfolio of projects I build at Microverse.
+
+Things done so far in the project:
+
+#### 1. Build mobile skeleton
+
+* Parse a Figma design
+* Use Flexbox for positioning
+* Use images and backgrounds
+
+## Built With
+
+- Major languages: HTML, CSS
+- Frameworks: none
```

Comment:
[ ] [Optional] You can remove this line to avoid confusion.

Reply:
Frameworks text has been removed.